bot.py

```diff
@@ -59,13 +59,19 @@ async def notify_subscribers(
     from_currency: str,
     to_currency: str,
     fees: float,
+    threshold: float,
 ):
     subscribers = await get_subscribers(session)
     logging.info(
         f"Notifying {len(subscribers)} subscribers about {from_currency} -> {to_currency} {swap_type} fees"
     )
 
-    message = f"Fees for {swap_type} {from_currency} -> {to_currency} at {encode_url_params(swap_type, from_currency, to_currency)}: {fees}%"
+    threshold_msg = (
+        f"just went below {threshold}%"
+        if fees < threshold
+        else f"above {threshold}% again"
+    )
+    message = f"Fees for {swap_type} {from_currency} -> {to_currency} at {encode_url_params(swap_type, from_currency, to_currency)} {threshold_msg}"
```

Comment:
```suggestion
    message = f"Fees for {swap_type} {from_currency} -> {to_currency} {threshold_msg}: {encode_url_params(swap_type, from_currency, to_currency)}"
```

Ugly URL in the end is nicer to read